packages/core/parcel-bundler/src/utils/loadSourceMap.js

```diff
@@ -2,12 +2,16 @@ const logger = require('@parcel/logger');
 const path = require('path');
 const fs = require('@parcel/fs');
 
-const SOURCEMAP_RE = /(?:\/\*|\/\/)\s*[@#]\s*sourceMappingURL\s*=\s*([^\s*]+)(?:\s*\*\/)?/;
+const SOURCEMAP_RE = /(?:\/\*|\/\/)\s*[@#]\s*sourceMappingURL\s*=\s*([^\s*]+)(?:\s*\*\/)?\s*$/;
```

Comment:
@mischnic I assume last changes on this regex were by you to make the css source mapping work. This change would only identify source maps at the end of the file. Hope this doesn't interfere with your changes?

Reply:
my change also added the `*/` comment end tag, matching whitespace after that is correct.